07_deploy_masters.sh

```diff
@@ -27,10 +27,10 @@ done
 openstack baremetal create $instack 
 mkdir -p configdrive/openstack/latest
 cp ocp/master.ign configdrive/openstack/latest/user_data
-for node in $(jq -r .nodes[].name $instack); do
+for node in $(jq -r .nodes[:1][].name $instack); do
 
   # FIXME(shardy) we should parameterize the image
-  openstack baremetal node set $node --instance-info image_source=http://172.22.0.1/images/redhat-coreos-maipo-47.284-openstack_dualdhcp.qcow2 --instance-info image_checksum=$(md5sum images/redhat-coreos-maipo-47.284-openstack_dualdhcp.qcow2 | awk '{print $1}') --instance-info root_gb=25 --property root_device="{\"name\": \"$ROOT_DISK\"}"
+  openstack baremetal node set $node --instance-info image_source=http://172.22.0.1/images/redhat-coreos-maipo-47.284-openstack_dualdhcp.qcow2 --instance-info image_checksum=$(md5sum $IRONIC_DATA_DIR/html/images/redhat-coreos-maipo-47.284-openstack_dualdhcp.qcow2 | awk '{print $1}') --instance-info root_gb=25 --property root_device="{\"name\": \"$ROOT_DISK\"}"
```

Comment:
```suggestion
  openstack baremetal node set $node \
      --instance-info image_source=http://172.22.0.1/images/redhat-coreos-maipo-47.284-openstack_dualdhcp.qcow2 \
      --instance-info image_checksum=$(md5sum "${IRONIC_DATA_DIR}/html/images/redhat-coreos-maipo-47.284-openstack_dualdhcp.qcow2" | awk '{print $1}') \
      --instance-info root_gb=25 --property root_device="{\"name\": \"$ROOT_DISK\"}"
```

Reply:
Gonna leave this as is for the moment as I havn't touched the file, just renamed it